src/Rector/PropertyFetch/OptionalToNullsafeOperatorRector.php

```diff
@@ -129,7 +129,7 @@ public function refactor(Node $node): ?Node
             return new NullsafePropertyFetch($firstArg->value, $node->name);
         }
 
-        return new NullsafeMethodCall($firstArg->value, $node->name, $node->args);
+        return new NullsafeMethodCall($firstArg->value, $node->name, $node->getArgs());
```

Comment:
As for these, it's better to ignore the PHPStan message in `phpstan.neon`.

For time being Rector and PHPStan use different php-parser, that leads to false positives that are impossible to fix.

Reply:
Copied the message from the https://github.com/rectorphp/rector-symfony/pull/32/files.